benchmark/infrastructure/BenchmarkMeasurementContainer.cpp

```diff
@@ -68,38 +68,35 @@ ResultGroup::operator std::string() const {
   std::ostringstream stream;
 
   /*
-  If the given vector is empty, add ` None` to the stream. If it isn't, adds
-  "\n\n" and then calls the given function.
+  If the given vector is empty, return ` None`. If it isn't, return the
+  contacination of "\n\n" and then the string list representation of the vector.
   */
-  auto addWithFunctionOrNone = [&stream](const auto& vec, auto func) {
-    if (!vec.empty()) {
-      stream << "\n\n";
-      func();
-    } else {
-      stream << " None";
-    }
+  auto vectorToStringListOrNone = []<typename T>(const std::vector<T>& vec) {
+    return (vec.empty()
+                ? " None"
```

Comment:
This lambda has way too many indents itself. Use an explicit `if empty -> return ""None".

Reply:
Done ,see [3ac0484](https://github.com/ad-freiburg/qlever/pull/1010/commits/3ac048457a6d4d0860d803c691a04cdadf732c08).